.github/ISSUE_TEMPLATE/add-plugin-to-marketplace.yml

```diff
@@ -13,7 +13,7 @@ body:
       value: |
         Thank you for building a plugin for Hermes and sharing it with the community!
 
-        The [Hermes plugin marketplace](https://hermes.software-metadata.pub#plugins) is a curated list of plugins on our website that allows you to share plugins that you developed, and others to find them.
+        The [Hermes plugin marketplace](https://hermes.software-metadata.pub/plugins/marketplace.html) is a curated list of plugins on our website that allows you to share plugins that you developed, and others to find them.
```

Comment:
The `MARKETPLACE_URL`, i.e. `https://hermes.software-metadata.pub/marketplace`, should be used here as well. I think when I originally created the template, this did not exist yet.